apple/default.nix

```diff
@@ -2,6 +2,7 @@
 
 {
   boot.kernelParams = [
+    "hid_apple.swap_opt_cmd=1"
```

Comment:
Given that it is not easy to disable those options again, should we rather have a `Readme.md` in `./apple` that just document this option?

```markdown
## Switching Cmd and Alt/AltGr

This will switch the left Alt and Cmd key as well as the right Alt/AltGr and Cmd key. 

boot.kernelParams = [
  "hid_apple.swap_opt_cmd=1"
];
```

Reply:
yeah, let's do that. updated.